development/fitness-functions/common/constants.ts

```diff
@@ -1,7 +1,7 @@
 // include JS, TS, JSX, TSX files only excluding files in the e2e tests and
 // fitness functions directories
 const EXCLUDE_E2E_TESTS_REGEX =
-  '^(?!test/e2e)(?!development/fitness).*.(js|ts|jsx|tsx)$';
+  '^(?!test/e2e)(?!development/fitness|development/webpack).*.(js|ts|jsx|tsx)$';
```

Comment:
I don't remember what these fitness functions do, but I do remember they were in the way and this made them stop yelling at me.